src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
```

Comment:
I think we can delete this stuff. No point carrying it around in every one of these files we create. But it's minor, I won't hold up the PR on this.